### DISCUSSION TITLE
Welcome to MyIP Discussions!

### DISCUSSION THREAD

**Discussion by jason5ng32** [jason5ng32/MyIP#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chengvincent1** [comment]
![1](https://github.com/user-attachments/assets/44071597-2e20-4f48-a29e-6ce12c42d804)
![2](https://github.com/user-attachments/assets/d30ab0a3-4b94-4985-9942-a49c6bcc3d74)
群晖的docker刚部署完，不能用- = 各位大佬们咋搞

**Comment by JReming85** [comment]
Can you have a page to check an IP against all the GeoIP APIs?

My main issue is confirming IP locations, and it seems some companies don't update it as often as others… cough cough microsoft

Or even let the user specify an IP for the front page.

**Reply by jason5ng32** [reply]
will consider
